src/_data/sites/zh-cn.yml

```diff
@@ -0,0 +1,515 @@
+#------------------------------------------------------------------------------
+# Chinese Simplified Site Details
+# The primary ESLint site that is hosted at eslint.org
+# Author: aladdin-add<weiran.zsd@outlook.com>
+#------------------------------------------------------------------------------
+
+
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: zh-cn
+  flag: 🇨🇳
+  name: 简体中文
+locale: zh-cn
+hostname: zh-cn.eslint.org
```

Comment:
Should this replace cn.eslint.org eventually?

Reply:
yes!

Reply:
Then maybe new.cn.eslint.org for now?